src/screens/HomeScreen.js

```diff
@@ -154,7 +168,16 @@ export const HomeScreen = ({ navigation }) => (
               title: 'Veranstaltung',
               query: 'eventRecord',
               queryVariables: { id: `${eventRecord.id}` },
-              rootRouteName: 'EventRecords'
+              rootRouteName: 'EventRecords',
+              shareContent: {
+                message: `[Bad Belzig App] ${momentFormat(
```

Comment:
Hier vielleicht ein "ToDo" dran für später: "Name kommt aus den Settings"

Reply:
ja stimmt! den namen hab ich schon hinterlegt in app settings file hier. kann ich daher nehmen. thx, gut gesehen.

Reply:
und direkt auch in einen helper refactored das bauen der `message`